### DISCUSSION TITLE
在下课时显示必须手动关闭的提示框

### DISCUSSION THREAD

**Discussion by HelloWRC** [ClassIsland/ClassIsland#554] [插件功能请求]
> ……
> 2. 可以在下课时右下角一个显眼且必须手动关闭的提框。

_Originally posted by @kaokao221 in https://github.com/HelloWRC/ClassIsland/issues/53#issuecomment-2031437699_

在下课时显示一个显眼且必须手动关闭的提示框。

**设计思路**

在17763以上使用Windows 内置的Toast功能显示一个永久开启且需要手动点击按钮确认关闭的Toast，在这个版本以下的系统使用自行实现的通知框。

**Comment by jiangyin14** [comment]
我认为没必要实现。这个框麻烦的是老师而不是学生 所以应该在不影响使用电脑的情况下 尽可能把提醒强度拉满
不过如果实现难度不大的话也可以做，按需启用嘛（

**Comment by jiangyin-srinternet** [comment]
翻看issue时再次表示dislike（

**Comment by FrankXiDong** [comment]
> 我认为没必要实现。这个框麻烦的是老师而不是学生 所以应该在不影响使用电脑的情况下 尽可能把提醒强度拉满 不过如果实现难度不大的话也可以做，按需启用嘛（

话说不就是为了麻烦老师，阻碍拖堂吗（误）

**Comment by Pro-Qin** [comment]
> > 我认为没必要实现。这个框麻烦的是老师而不是学生 所以应该在不影响使用电脑的情况下 尽可能把提醒强度拉满 不过如果实现难度不大的话也可以做，按需启用嘛（
> 
> 
> 
> 话说不就是为了麻烦老师，阻碍拖堂吗（误）

被老师严令禁止就老实了🤓
